app/helpers/article_helper.rb

```diff
@@ -71,6 +77,37 @@ def render_text_from_html(values)
     end.flatten
   end
 
+  #
+  def transform_research_starter_text(options = {})
```

Comment:
I'm wondering if this might be useful to be in a presenter.  It will probably be a bit easier to reason about/test in a class as opposed to a helper method.